api-samples/tabs/inspector/tabs_api.js

```diff
@@ -0,0 +1,412 @@
+  refreshWindow(window.id);
+});
+
+chrome.windows.onFocusChanged.addListener(function (windowId) {
+  focusedWindowId = windowId;
+  appendToLog('windows.onFocusChanged -- window: ' + windowId);
+  loadWindowList();
+});
+
+chrome.windows.onRemoved.addListener(function (windowId) {
+  appendToLog('windows.onRemoved -- window: ' + windowId);
+  loadWindowList();
+});
+
+chrome.tabs.onCreated.addListener(function (tab) {
+  appendToLog(
+    'tabs.onCreated -- window: ' +
+      tab.windowId +
+      ' tab: ' +
+      tab.id +
+      ' title: ' +
+      tab.title +
+      ' index ' +
+      tab.index +
+      ' url ' +
+      tab.url
+  );
+  loadWindowList();
+});
+
+chrome.tabs.onAttached.addListener(function (tabId, props) {
+  appendToLog(
+    'tabs.onAttached -- window: ' +
+      props.newWindowId +
+      ' tab: ' +
+      tabId +
+      ' index ' +
+      props.newPosition
+  );
+  loadWindowList();
+});
+
+chrome.tabs.onMoved.addListener(function (tabId, props) {
+  appendToLog(
+    'tabs.onMoved -- window: ' +
+      props.windowId +
+      ' tab: ' +
+      tabId +
+      ' from ' +
+      props.fromIndex +
+      ' to ' +
+      props.toIndex
+  );
+  loadWindowList();
+});
+
+async function refreshTab(tabId) {
+  const tab = await chrome.tabs.get(tabId);
+  const output = document.getElementById('tab_' + tab.id);
+  if (!output) return;
+  renderTab(tab, output);
+  appendToLog('tab refreshed -- tabId: ' + tab.id + ' url: ' + tab.url);
+}
+
+chrome.tabs.onUpdated.addListener(function (tabId, props) {
+  appendToLog(
+    'tabs.onUpdated -- tab: ' +
+      tabId +
+      ' status ' +
+      props.status +
+      ' url ' +
+      props.url
+  );
+  refreshTab(tabId);
+});
+
+chrome.tabs.onDetached.addListener(function (tabId, props) {
+  appendToLog(
+    'tabs.onDetached -- window: ' +
+      props.oldWindowId +
+      ' tab: ' +
+      tabId +
+      ' index ' +
+      props.oldPosition
+  );
+  loadWindowList();
+});
+
+chrome.tabs.onActivated.addListener(function (props) {
+  appendToLog(
+    'tabs.onActivated -- window: ' + props.windowId + ' tab: ' + props.tabId
+  );
+  loadWindowList();
+});
+
+chrome.tabs.onRemoved.addListener(function (tabId) {
+  appendToLog('tabs.onRemoved -- tab: ' + tabId);
+  loadWindowList();
+});
+
+async function createWindow() {
+  const args = {
+    left: parseInt(document.getElementById('new_window_left').value),
+    top: parseInt(document.getElementById('new_window_top').value),
+    width: parseInt(document.getElementById('new_window_width').value),
+    height: parseInt(document.getElementById('new_window_height').value),
+    url: document.getElementById('new_window_url').value
+  };
+
+  if (!isInt(args.left)) delete args.left;
+  if (!isInt(args.top)) delete args.top;
+  if (!isInt(args.width)) delete args.width;
+  if (!isInt(args.height)) delete args.height;
+  if (!args.url) delete args.url;
+
+  chrome.windows.create(args).catch(alert);
+}
+
+document
```

Comment:
Is there a reason why this isn't further down and using the same style as all of the others?

Reply:
I think the code's style is consistent with the code below:
https://github.com/GoogleChrome/chrome-extensions-samples/pull/964/files#diff-72263fa8715a4af88574ae6eeb6a670e8d95b158a0b6baba8fefa8941b7ffdebR168-R179